cEP-0006.md

```diff
@@ -87,3 +83,71 @@ Original text courtesy of the [Speak Up! project] (http://web.archive.org/web/20
 
 If you have questions, feel free to contact us by emailing `community AT
 coala DOT io`.
+
+
+Reporting a Code of Conduct Violation
+-------------------------------------
+
+If you believe someone is violating the code of conduct we ask that you report
+it by creating an issue at https://gitlab.com/coala-coc/coc-issues, and check
+the "This issue is confidential and should only be visible to team members with
+at least Reporter access." checkbox.  We strive to keep all issues confidential.
+
```

Comment:
I'm not sure only the community team leader can read the community mails so this would be a privacy problem.